debian/control

```diff
@@ -0,0 +1,28 @@
+Source: bosh-agent
+Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+XSBC-Original-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
+Uploaders: Christian Ehrhardt <christian.ehrhardt@canonical.com>
+Section: devel
+Testsuite: autopkgtest-pkg-go
+Priority: optional
+Build-Depends: debhelper,
+               debhelper-compat (= 11),
+               dh-golang,
+               golang-1.13-go
+Standards-Version: 4.1.4
+Vcs-Browser: https://salsa.debian.org/go-team/packages/bosh-agent
+Vcs-Git: https://salsa.debian.org/go-team/packages/bosh-agent.git
```

Comment:
We should probably put the repository somewhere on LP and put those links here, instead. There are wrong URLs, hehe.

Reply:
Should you be able to point me to the correct place, please?